### PR TITLE
Fixes for Cottontail DB Adapter

### PR DESCRIPTION
**Pull request by ppanopticon**
Addresses #255, #256, #257 #258, #257 as requested by @Spiess

### CODE REVIEW

**Comment by ppanopticon** [comment]
These fixes should address the issues reported by @Spiess 

I'm not if removing ADAMpro should als be included in this PR.

**Comment by Spiess** [comment]
Thank you for the quick fixes. Once you have confirmed that the many database channels are indeed intended on #258 I will approve the mere.

File: cineast-api/src/main/java/org/vitrivr/cineast/api/Main.java [hunk 1]
```diff
@@ -27,12 +27,26 @@ public static void main(String[] args) {
...
+    /* Start API endpoint. */
+    try {
+      APIEndpoint.getInstance().start();
+    } catch (Throwable e) {
+      System.err.println("Failed to initialize API endpoint due to an exception: " + e.getMessage());
+    }
+
+    /* Start gRPC endpoint. */
+    try {
+      GRPCEndpoint.start();
```
Comment (lucaro) [inline cineast-api/src/main/java/org/vitrivr/cineast/api/Main.java:39]: If you do error checking here, the block in https://github.com/vitrivr/cineast/blob/cecc4ac9c69f4806a1ab04d9c9888d432fd6eb49/cineast-api/src/main/java/org/vitrivr/cineast/api/GRPCEndpoint.java#L36 can be removed (and should be, as failing silently is never a good strategy).

**Review by lucaro — CHANGES_REQUESTED** [review]

File: cineast-api/src/main/java/org/vitrivr/cineast/api/Main.java (hunk 1 above)
Comment (ppanopticon) [reply cineast-api/src/main/java/org/vitrivr/cineast/api/Main.java:39]: Done!

**Comment by ppanopticon** [comment]
> Thank you for the quick fixes. Once you have confirmed that the many database channels are indeed intended on #258 I will approve the merge.

This is intended. The CottontailWrapper class is lightweight and re-created many times. The only thing I'm not sure about is whether pinging the endpoint is necessary every, single time.

**Review by lucaro — APPROVED** [review]

**Pull request merged**